.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Deploy Debezium Website
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: docker run --rm --privileged=true --user=$UID:$UID -i -p 4242:4242 -e LC_ALL=C.UTF-8 -e LANG=C.UTF-8 -v $(pwd):/site -v ~/github/debezium:/debezium debezium/awestruct bash
+
+      - name: Deploy
```

Comment:
Awestruct handles the deployment itself, so this step isn't required.  We may need to use this when we swap to Hugo, that's still TBD; so you may wish to simply comment this out of the file temporarily.

Reply:
Will be commenting those for now! Thnx.